hardware/arduino/avr/boards.txt

```diff
@@ -288,8 +288,8 @@ leonardo.bootloader.low_fuses=0xff
 leonardo.bootloader.high_fuses=0xd8
 leonardo.bootloader.extended_fuses=0xcb
 leonardo.bootloader.file=caterina/Caterina-Leonardo.hex
-leonardo.bootloader.unlock_bits=0x3F
-leonardo.bootloader.lock_bits=0x2F
+leonardo.bootloader.unlock_bits=0xFF
+leonardo.bootloader.lock_bits=0xCF
```

Comment:
Thanks for pointing this out.
I believed that protecting bootloader with mode3 (like all the other boards) shouldn't have any side effect but I was wrong, reading the bootloader section from application space is used here https://github.com/arduino/Arduino/blob/master/hardware/arduino/avr/cores/arduino/USBCore.cpp#L819
Reverting these fuses to 0xEF in next commit